pkg/utils/restcfg/ratelimiting.go

```diff
@@ -38,18 +42,21 @@ func InitFlags(flagset *flag.FlagSet) {
 		flagset = flag.CommandLine
 	}
 
+	flagset.DurationVar(&timeout, "client-timeout", timeout,
+		"The maximum length of time to wait before giving up on a server request. A value of zero means no timeout.")
 	flagset.UintVar(&qps, "client-qps", qps, "The maximum number of queries per second performed towards the API server.")
-	flagset.UintVar(&burst, "client-max-burst", qps, "The maximum burst of requests in excess of the rate limit towards the API server.")
+	flagset.UintVar(&burst, "client-max-burst", burst, "The maximum burst of requests in excess of the rate limit towards the API server.")
```

Comment:
I think the default values should be `DefaultClientTimeout`, `DefaultQPS` and `DefaultBurst`

Reply:
I know amd I agree with you, but if you look at the beginning of that file qps and burst are initialized with DefaultBurst and DefaultQPS. I know it's strange but it works. If you want I can change it

Reply:
What about changing those values (keeping the suggestion from @aleoli ) and see what happens?

Reply:
Ok let's clean the code